services/src/main/java/io/druid/cli/validate/DruidJsonValidator.java

```diff
@@ -73,7 +86,23 @@ public void run()
       System.out.printf("File[%s] does not exist.%n", file);
     }
 
-    final ObjectMapper jsonMapper = new DefaultObjectMapper();
+    final Injector injector = GuiceInjectors.makeStartupInjector();
```

Comment:
You can see where the GuiceRunnable approach is used `inio.druid.cli.CliPeon#run` You'll have to define a Guice Module like 


```java
  @Override
  protected List<? extends Module> getModules()
  {
    return ImmutableList.<Module>of(
        new Module()
        {
          @Override
          public void configure(Binder binder)
          {
            binder.bindConstant().annotatedWith(Names.named("serviceName")).to("druid/jsonvalidator");
            binder.bindConstant().annotatedWith(Names.named("servicePort")).to(-1);
          }
        }
      }
```